collection.json

```diff
@@ -1 +1 @@
-[{"collectionId":"93a743c7-8dcd-4c86-bc9d-11d50fc1fd41","title":"Art Marketplace","img":"images/collections/collection_1648122731244.png"},{"collectionId":"566d9157-fdf1-4abe-8130-2d41a821006d","title":"Collection 1","img":"images/collections/collection_1648122713038.png"},{"collectionId":"70a7f4e2-6923-4e74-a6cd-db23be96a695","title":"Collection 2","img":"images/collections/collection_1648122661590.png"},{"collectionId":"d2de3169-e825-44d6-96ad-46d3397c89c2","title":"Collection 3","img":"images/collections/collection_1648132021107.png"},{"collectionId":"963d239b-2c96-4d95-a96b-f7396eda8bda","title":"Minions","img":"images/collections/collection_1648568775374.png"}]
\ No newline at end of file
+[{"collectionId":"93a743c7-8dcd-4c86-bc9d-11d50fc1fd41","title":"Art Marketplace","img":"images/collections/collection_1648122731244.png"},{"collectionId":"566d9157-fdf1-4abe-8130-2d41a821006d","title":"Collection 1","img":"images/collections/collection_1648122713038.png"},{"collectionId":"70a7f4e2-6923-4e74-a6cd-db23be96a695","title":"Collection 2","img":"images/collections/collection_1648122661590.png"},{"collectionId":"d2de3169-e825-44d6-96ad-46d3397c89c2","title":"Collection 3","img":"images/collections/collection_1648132021107.png"},{"collectionId":"963d239b-2c96-4d95-a96b-f7396eda8bda","title":"Minions","img":"images/collections/collection_1648568775374.png"},{"collectionId":"fde2e7e0-c1e6-4de3-81fb-02771e11c30f","title":"Temp Collection","img":"images/collections/collection_1653703618141.png"}]
```

Comment:
What is this for? Why is data stored here?

Reply:
It is not necessary.  I had to remove it.